WalletWasabi.Gui/Global.cs

```diff
@@ -172,10 +172,19 @@ public async Task InitializeNoWalletAsync(TerminateService terminateService)
 
 				#region BitcoinStoreInitialization
 
-				await bstoreInitTask.ConfigureAwait(false);
+				try
+				{
+					await bstoreInitTask.ConfigureAwait(false);
```

Comment:
In https://github.com/nopara73/WalletWasabi/blob/caf8d6058813936fcc0dbf31d703f1be4ac2f3ed/WalletWasabi/Blockchain/Transactions/TransactionStore.cs#L99 a very specific exception must be thrown and we must make sure it can differentiate between mempool transaction file corruption and confirmed transaction file corruption.